### ISSUE TITLE
Add support for testing code which needs to run in background function

### ISSUE THREAD

**Issue by t-b** [byte-physics/igor-unit-testing-framework#29]
In the tests for https://github.com/AllenInstitute/MIES we have the need to test data acquisition which runs in a background function. I've hacked a solution together but that is, well, hacky. See `run` in `UTF_HardwareMain.ipf`.

Currently we have two test cases, the first sets up DAQ and starts it and the second does the tests. The background function `WaitUntilDAQDone_IGNORE` starts the second test case after DAQ has finished.

Current issues with that approach:
- It is outside of the UTF
- It creates one JUNIT xml file for each test case
- We can't use autodiscovery of test cases but have to manually list them
- Fatal errors in one of the test cases don't stop the full run

A proper support would entail:
- one JUNIT XML file per Igor experiment
- full autodiscovery support
- user configurable condition when the second test case needs to be run

**Comment by t-b** [comment]
These two testcases are run in order, runTest stops after the first and continues after the listed bkg functions to `RegisterToBeMonitoredBkgFunctions` are done via a monitoring background task.

```
Function TPDuringDAQOnlyTP_1()

	STRUCT DAQSettings s
	InitDAQSettingsFromString(s, "DAQ_MD1_RA0_IDX0_LIDX0_BKG_1_RES_1")
	AcquireData(s, preAcquireFunc=TPDuringDAQOnlyTP_IGNORE)
	
	RegisterToBeMonitoredBkgFunctions("Testpulse;TestpulseMD;...", mode = <AND or OR>)
End

Function TPDuringDAQOnlyTP_2()
End
```

One possible solution:
- New flag for runtest to stop/continue its run
- New flag for finding test cases with numeric suffix which will then be run in that order

**Comment by t-b** [comment]
Closed via #67.
